src/fleet/azext_fleet/custom.py

```diff
@@ -63,7 +63,7 @@ def update_fleet(cmd,
         operation_group="fleets"
     )
     fleet_patch = fleet_patch_model(tags=tags)
-    return client.update(resource_group_name, name, None, fleet_patch)
+    return client.update(resource_group_name, name, fleet_patch, None)
```

Comment:
is the order of the parameters in the python sdk dependent on how they appear in the swagger file?
that is quite annoying...

Is there a way to specify/enforce the order on the sdk generation side with some autorest decorations?